elm/pipeline/steps.py

```diff
@@ -55,12 +44,20 @@ class Wrapped(SklearnMixin, cls):
     for cls in get_module_classes(m).values():
         if cls.__name__ in _seen:
             continue
+        if not m in cls.__module__:
```

Comment:
This is just checking that we are getting `StandardScaler` or similar from the `sklearn` module where it is actually defined, not some other one where it is imported for internal usage.